SimPEG/data_misfit.py

```diff
@@ -8,12 +8,10 @@
 
 
 class BaseDataMisfit(L2ObjectiveFunction):
```

Comment:
Base classes are a good place to add descriptive docstring for what the .py file actually has. And we need to define the input parameters.